### PR TITLE
chore(prop-types): add proptypes to Grid and Cell

### PR DESCRIPTION
**Pull request by James-E-Adams**
Add proptypes.

### CODE REVIEW

File: lib/Cell.js [hunk 1]
```diff
@@ -24,4 +25,13 @@ const Cell = styled.section`
...
 `;
 
+Cell.propTypes = {
+  width: PropTypes.number,
+  height: PropTypes.number,
+  top: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  left: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  middle: PropTypes.bool,
+  center: PropTypes.bool,
+  area: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
```
Comment (azz) [inline lib/Cell.js]: This one is just `PropTypes.string`

File: package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "normalize.css": "^7.0.0",
     "prettier": "^1.5.3",
     "prismjs": "^1.6.0",
+    "prop-types": "^15.6.0",
```
Comment (azz) [inline package.json]: This is a dependency, not a `devDependency`.

File: lib/Cell.js (hunk 1 above)
Comment (James-E-Adams) [reply lib/Cell.js]: Yup, fair enough. Will fix.

File: package.json (hunk 2 above)
Comment (James-E-Adams) [reply package.json]: Yeah - I wasn't sure about that.

**Comment by James-E-Adams** [comment]
Changes as requested.

**Comment by azz** [comment]
Thanks 🌷

**Comment by James-E-Adams** [comment]
:rose:

**Pull request merged**